shardingsphere-features/shardingsphere-readwrite-splitting/shardingsphere-readwrite-splitting-core/src/main/java/org/apache/shardingsphere/readwritesplitting/rule/ReadwriteSplittingRule.java

```diff
@@ -127,18 +128,19 @@ public void updateStatus(final DataSourceStatusChangedEvent event) {
     
     @Override
     public Map<String, Supplier<Object>> getExportedMethods() {
-        Map<String, Supplier<Object>> result = new HashMap<>(3, 1);
-        result.put(ExportableConstants.EXPORTABLE_KEY_AUTO_AWARE_DATA_SOURCE, () -> exportDataSource(DYNAMIC));
+        Map<String, Supplier<Object>> result = new HashMap<>(4, 1);
+        result.put(ExportableConstants.EXPORTABLE_KEY_AUTO_AWARE_DATA_SOURCE, () -> exportDataSource(false, DYNAMIC));
         result.put(ExportableConstants.EXPORTABLE_KEY_AUTO_AWARE_DATA_SOURCE_NAME, this::exportAutoAwareDataSourceNames);
-        result.put(ExportableConstants.EXPORTABLE_KEY_DATA_SOURCE, () -> exportDataSource(STATIC));
+        result.put(ExportableConstants.EXPORTABLE_KEY_ENABLED_DATA_SOURCE, () -> exportDataSource(true, STATIC));
+        result.put(ExportableConstants.EXPORTABLE_KEY_DATA_SOURCE, () -> exportDataSource(false, DYNAMIC, STATIC));
         return result;
     }
     
-    private Map<String, Map<String, String>> exportDataSource(final String readwriteSplittingType) {
-        Map<String, Map<String, String>> result = new HashMap<>(dataSourceRules.size(), 1);
+    private Map<String, Map<String, String>> exportDataSource(final boolean removeDisabled, final String... readwriteSplittingType) {
```

Comment:
`readwriteSplittingType` this variable dones't sounds like an array. what does the contant of this variable like

Reply:
it might need a better name